src/main/zapHomeFiles/hud/libs/i18n.js

```diff
@@ -0,0 +1,256 @@
+import Vue from "vue";
+import VueI18n from "vue-i18n";
+
+const messages = {
+  en_GB: {
```

Comment:
I have a lot of comments about this i18n approach.
- It will not scale if you suddenly have dozens of languages and unnecessarily bloats the Vue object.
- There is no-lazy loading possible with this approach. 
- JS to store strings opens up direct "false friend" translation injection attacks, aka `eval()` / `constructor.constructor()` etc.
- including HTML requires using v-html, which is a very well known XSS gateway.
- if you MUST use v-html, use freeze to harden these strings, because at runtime they will never change

Kazupon's lib does a great job of protecting Vue from attack vectors (even better than Vue itself, I know - I helped him harden it), but since translations are generally not as closely audited as code blocks, it is trivial to smuggle in <img> and <object> tags that can of course do really bad things. With the intended audience this is not as big an issue as it would be if your toddler was using the interface on an iPad, but best practice should be followed here in all cases.